lib/shower/shower.js

```diff
@@ -200,15 +200,15 @@ class Shower {
     }
 
     _initLiveRegion() {
-        const lr = document.createElement('section');
-        lr.className = 'region';
-        lr.setAttribute('role', 'region');
-        lr.setAttribute('aria-live', 'assertive');
-        lr.setAttribute('aria-relevant', 'additions');
-        lr.setAttribute('aria-label', 'Slide Content: Auto-updating');
-
-        this._liveRegion = lr;
-        document.body.appendChild(lr);
+        const region = document.createElement('section');
```

Comment:
Long variable names look extremely noisy in some cases. But whatever, as you wish.